op.go

```diff
@@ -299,3 +300,12 @@ func decodeBigIntHook(f reflect.Kind, t reflect.Kind, data interface{}) (interfa
 
 	return data, nil
 }
+
+func sortedKeys(m map[string]interface{}) []string {
+	var keys []string
```

Comment:
You already know how large this slice needs to be, so you can make this more efficient:
```suggestion
	keys := make([]string, 0, len(m))
```